router/router.go

```diff
@@ -67,7 +68,8 @@ import (
 	"github.com/openziti/ziti/router/xgress_transport_udp"
 	"github.com/openziti/ziti/router/xlink"
 	"github.com/openziti/ziti/router/xlink_transport"
-	"github.com/pkg/errors"
+	pkgerrors "github.com/pkg/errors"
```

Comment:
This dep should be removed, and we should rely on the "errors" package. Also, why was an alliase introduced?

`.Is` to `.As`  in the "errors' package.

Reply:
I see why, you have the "errors" package as well. I'd just remove this one.